packages/chrome/src/elements/body/Body.tsx

```diff
@@ -5,27 +5,27 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { useMemo } from 'react';
-import PropTypes from 'prop-types';
-import { IBodyProps } from '../../types';
+import React, { HTMLAttributes, useMemo, useState } from 'react';
 import { StyledBody } from '../../styled';
 import { BodyContext } from '../../utils/useBodyContext';
 
 /**
  * @extends HTMLAttributes<HTMLDivElement>
  */
-export const Body = React.forwardRef<HTMLDivElement, IBodyProps>(({ hasFooter, ...props }, ref) => {
-  const bodyContextValue = useMemo(() => ({ hasFooter: !!hasFooter }), [hasFooter]);
+export const Body = React.forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
+  (props, ref) => {
+    const [hasFooter, setHasFooter] = useState(true);
```

Comment:
Shouldn't this initialize to `useState(false)`?

Reply:
I'm retaining the previous behavior which is [true by default](https://github.com/zendeskgarden/react-components/blob/next/packages/chrome/src/utils/useBodyContext.ts#L15). I'm OK shifting to `false` if that makes more sense though. EIther way the initial state has a 50/50 chance to not match the instructed UI. 🤔

Reply:
Weird. I think the previous was wrong. The `<Body>` definitely doesn't have a footer .. until it does 😅. There are plenty of areas that are `<Footer>`-less. Let's make sure the layout doesn't clip/overflow for any of these cases.

Reply:
I agree. Makes more sense to retain a "no footer" base case.